serveradmin/common/utils.py

```diff
@@ -28,3 +31,18 @@ def random_alnum_string(length):
         random_str = b64encode(random_bytes, b'??').decode().replace('?', '')
         if len(random_str) >= length:
             return random_str[:length]
+
+
+def profile(func):
+    @functools.wraps(func)
+    def wrapper_profile(*args, **kwargs):
+        start = time.perf_counter()
+        value = func(*args, **kwargs)
+        end = time.perf_counter()
+
+        duration = end - start
+        msg = 'Signal %s processed in %s seconds' % (func.__name__, duration)
```

Comment:
I find the wording `Signal` weird here. Something like `Function processing` or the like would make more sense to me.